roles/kubernetes/postinstall/tasks/rotate_kubelet_certs.yml

```diff
@@ -0,0 +1,24 @@
+
+- block:
+  - name: Turn on SERVING certificates rotation for kubelet
```

Comment:
This should only apply to master server?

Reply:
Right.